src/tools/tidy/src/ui_tests.rs

```diff
@@ -10,7 +10,7 @@ use std::path::{Path, PathBuf};
 
 const ENTRY_LIMIT: usize = 900;
 // FIXME: The following limits should be reduced eventually.
```

Comment:
I had always assumed that these numbers of files were unwieldy, but I didn't know about the GitHub 1000 file limit.

Can you move this comment up by one line and expand it to explain things better? Something like this:
```
FIXME: GitHub's UI truncates file lists that exceed 1000 entries, so these 
should all be 1000 or lower. Limits significantly smaller than 1000 are also
desirable, because large numbers of files are unwieldy in general. See issue
#73494.
```